packages/apron/apron.v0.9.14~beta.2/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "APRON numerical abstract domain library"
+authors: [
+  "Bertrand Jeannet"
+  "Antoine Miné"
+  "https://github.com/antoinemine/apron/graphs/contributors"
+]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://antoinemine.github.io/Apron/doc/"
+maintainer: "Nicolas Berthier <m@nberth.space>"
+dev-repo: "git+https://github.com/antoinemine/apron.git"
+bug-reports: "https://github.com/antoinemine/apron/issues"
+build: [
+  [
+    "sh"
+    "./configure"
+    "--prefix"
+    "%{share}%/apron"
+    "--no-ppl" {!conf-ppl:installed}
+    "--no-pplite" {!conf-pplite:installed}
+    "--no-ocaml-plugins" {os = "freebsd"}
+    "--absolute-dylibs" {os = "macos"}
+    "--ext-dll" {os = "win32"} "dll" {os = "win32"}
+    "--debug"
+    "--no-strip"
+  ]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "ocamlfind" {build}
+  "camlidl"
+  "mlgmpidl"
+  "ocamlbuild" {build}
+  "conf-perl"
+]
+depopts: [
+  "conf-ppl"
+  "conf-pplite"
```

Comment:
There's no conf-pplite package in opam-repository though.

Reply:
> There's no conf-pplite package in opam-repository though.

Yes. This was copied from ppl. Given that (unlike ppl) there's no pplite package for standard distributions (that I know of), adding a `conf-pplite` package may not be very useful. I suggest we remove this, and let the `./configure` script decide if pplite should be compiled in. What do you think?

Reply:
A conf-pplite package might not be that unreasonable because the `./configure` check might be close to how packages like conf-ppl do the check as well. Not having system packages and depexts maybe isn't an issue for having a conf-* package.

opam-repository maintainers probably know the best how they'd like things. Seems like opam and the CI don't really care that a depopt doesn't exist at all, so maybe it's fine as-is. If a conf-pplite package is created, then it'll also configure this package.

Reply:
I am fine with this, but I'd like to have the `conf-pplite` package as well, even if for the time being the list of depexts is empty. I think you can base it on https://github.com/ocaml/opam-repository/blob/master/packages/conf-ppl/conf-ppl.1/opam (or even simpler using conf-pkg-config if ppxlite supports pkg-config)

Reply:
As far as I know, pplite has no package installer from standard distribution, so, it's quite different from ppl. I think it is better to remove the dependency on conf-pplite until there is some more reason to include such a package.